build/proj/CoreBuild.proj

```diff
@@ -1,21 +1,18 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
-<Project DefaultTargets="LogInitialMessage;Restore;Build;GenerateDependentAssemblyVersionsFile;Sign;GenerateVSManifest" LocalProperty="SolutionPath">
+<Project DefaultTargets="LogInitialMessage;Restore;Build;GenerateDependentAssemblyVersionsFile;GenerateVSManifest" LocalProperty="SolutionPath">
 
   <!--
   Required parameters:
     Configuration                   Build configuration: "Debug", "Release", etc.
-                                    
   Optional parameters:              
     CIBuild                         "true" when building on CI server
     Build                           "true" to build solution
     Rebuild                         "true" to rebuild solution
     Deploy                          "true" to deploy assets (e.g. VSIXes)
     Test                            "true" to run tests
     IntegrationTest                 "true" to run integration tests
-    Sign                            "true" to sign built binaries
-    SignType                        "real" to send binaries to signing service, "test" to only validate signing configuration.
```

Comment:
The `Sign` target was removed entirely as MicroBuild signing is simply based on the plugin being present or not. In CI, the plugin gets installed which enables signing. It would be possible to put a toggle for signing as a pipeline variable, but it certainly has nothing to do with being a build flag at this point.

Reply:
Is "test" signing an option?

Reply:
Ah, yes! Thank you for reminding me. Sorry, a lot of stuff to keep track of.

What I can do is add a pipeline variable that sets the sign type. This is what CPS does in their pipeline:
![image](https://user-images.githubusercontent.com/17788297/151463076-af88d93f-4444-4994-a0dc-71f285a90ef7.png)

So, we can change that value that way. As it stands, I didn't modify our pipeline, but I can add an extra variable for this with no impact to our current builds.

Reply:
Ah ha! Did some research and it seems the preferred method for something like this is [runtime parameters](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/runtime-parameters?view=azure-devops&tabs=script) in the yaml file. I've done this and here's what it looks like:

![image](https://user-images.githubusercontent.com/17788297/151640774-b0d38f2c-11b4-48b2-a5a8-a2f8deaf8c28.png)

The default is `real` so we should be good to go. If you want to do a manual run of `test`, select it and it'll change the value when it installs the MicroBuild signing plugin.